less/home-page.less

```diff
@@ -1 +1,118 @@
-// Home Content
\ No newline at end of file
+// Home Content
+
+.container {
+  .flex(column; center; center);
+  width: 90%;
+  max-width: 880px;
+  margin: auto;
+  line-height: 1.5;
+
+  @media @phone {
+    width: 100%;
+  }
+
+  .homeHeaderPic {
+    width: 100%;
+  }
+
+  .homeLead {
+    padding: 15px;
+    .flex(column; space-evenly; flex-start);
```

Comment:
Great job following the recommended syntax here by separating the parameters with semicolons. 👍